src/components/Navbar.js

```diff
@@ -12,7 +12,7 @@ function Navbar() {
           <li><NavLink to="/"><FaArrowLeft /></NavLink></li>
           <li>WorldWide Covid Statistics</li>
           <li className="mic-settings">
-            <div><FaMicrophone /></div>
+            <div><FaSearch /></div>
             <div><HiCog /></div>
           </li>
         </ul>
```

Comment:
- [ ] Please implement the filtering function as requested by the previous reviewers in order to get approval. 🙏